### PR TITLE
[SwiftLint] Add Swiftlint

### PR DESCRIPTION
**Pull request by anayini**
## Changes in this pull request

Adds SwiftLint pod and fixes some errors.

Issue fixed: #7 (but doesn't yet call this from Danger)

### Checklist
- [x] All tests pass. Demo project builds and runs.

### CODE REVIEW

**Comment by anayini** [comment]
@jessesquires - Few open linter issues that I'm still trying to figure out, but will fix in a follow up.  This general structure should also allow me to hook up the danger-swiftlint integration.

File: Podfile.lock [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
 SPEC CHECKSUMS:
   Dwifft: bb97bff975a5f5e62a7492dc29463e9be47dc121
+  SwiftLint: 2aa21b96c8d13396a051bc1cb659d2ce1e0075b0
 
-PODFILE CHECKSUM: 9995c49f565033c41c2e8e49b97edfe9f4d2e713
+PODFILE CHECKSUM: 0bfb80dc9c70032913e17c7e6eb4c95e65c8544c
 
-COCOAPODS: 1.4.0
+COCOAPODS: 1.1.1
```
Comment (Ben-G) [inline Podfile.lock]: Looks like your system CocoaPods is a little outdated? 😉

**Comment by Ben-G** [comment]
@anayini Generally looks good, but much of this diff stems from using an old CocoaPods version. Can you update your system version to 1.4.0, then re-run `pod install`?

**Comment by anayini** [comment]
Oops thought I had done that - yes will do.

**Pull request merged**